Apps/Sandcastle/gallery/3D Tiles Point Cloud Attenuation and Eye Dome Lighting.html

```diff
@@ -0,0 +1,186 @@
+            <input type="text" size="5" data-bind="value: baseResolution">
+        </td>
+    </tr>
+    <tr><td class="header">Eye Dome Lighting</td></tr>
+    <tr>
+        <td>Eye Dome Lighting Strength</td>
+        <td>
+            <input type="range" min="0.0" max="10.0" step="0.1" data-bind="value: eyeDomeLightingStrength, valueUpdate: 'input'">
+            <input type="text" size="5" data-bind="value: eyeDomeLightingStrength">
+        </td>
+    </tr>
+    <tr>
+        <td>Eye Dome Lighting Radius</td>
+        <td>
+            <input type="range" min="0.0" max="10.0" step="0.1" data-bind="value: eyeDomeLightingRadius, valueUpdate: 'input'">
+            <input type="text" size="5" data-bind="value: eyeDomeLightingRadius">
+        </td>
+    </tr>
+    </tbody></table>
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    baseLayerPicker : false
+});
+
+var scene = viewer.scene;
+
+// Mt. St. Helens 3D Tileset generated from LAS provided by https://www.liblas.org/samples/
+var url = 'https://beta.cesium.com/api/assets/3523?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIzNDBlZDJhZS1jMDlmLTRiNWEtODhiMC1hMjExNGUzYmY5NzYiLCJpZCI6MjUsImlhdCI6MTQ4NTQ1OTAwMn0.PJQ7H5FsfSPBpQ2-_0jYNI4FnJBwuNpXnqfa7MZXFtc';
+var tileset = scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : url
+}));
+
+tileset.readyPromise.then(function() {
+    var boundingSphere = tileset.boundingSphere;
+    viewer.camera.viewBoundingSphere(boundingSphere, new Cesium.HeadingPitchRange(0.0, -1.0, boundingSphere.radius));
+    viewer.camera.lookAtTransform(Cesium.Matrix4.IDENTITY);
+    tileset.maximumScreenSpaceError = 16.0;
+});
+
+// The viewModel tracks the state of our mini application.
+var viewModel = {
+    maximumScreenSpaceError : 16.0,
+    geometricErrorScale : 1.0,
+    maximumAttenuation : 0, // Equivalent to undefined
+    baseResolution : 0, // Equivalent to undefined
+    eyeDomeLightingStrength : 1.0,
+    eyeDomeLightingRadius : 1.0
+};
+
+function checkZero(newValue) {
+    var newValueFloat = parseFloat(newValue);
+    return (newValueFloat === 0.0) ? undefined : newValueFloat;
+}
+
+// Convert the viewModel members into knockout observables.
+Cesium.knockout.track(viewModel);
+
+// Bind the viewModel to the DOM elements of the UI that call for it.
+var toolbar = document.getElementById('toolbar');
+Cesium.knockout.applyBindings(viewModel, toolbar);
+
+Cesium.knockout.getObservable(viewModel, 'maximumScreenSpaceError').subscribe(
+    function(newValue) {
+        tileset.maximumScreenSpaceError = parseFloat(newValue);
+    }
+);
+
+Cesium.knockout.getObservable(viewModel, 'geometricErrorScale').subscribe(
+    function(newValue) {
+        tileset.pointAttenuationOptions.geometricErrorScale = parseFloat(newValue);
```

Comment:
Not sure about the "options" suffix for `pointAttenuationOptions`.

Perhaps `pointShading`?

Reply:
This will also make `eyeDomeLightingStrength` and `eyeDomeLightingRadius` make more sense since it doesn't seem - at least from the user's perspective - that they should be a property on an attenuation object.